src/source/Crypto/Tls_mbedtls.c

```diff
@@ -26,9 +28,17 @@ STATUS createTlsSession(PTlsSessionCallbacks pCallbacks, PTlsSession* ppTlsSessi
     mbedtls_ssl_config_init(&pTlsSession->sslCtxConfig);
     mbedtls_ssl_init(&pTlsSession->sslCtx);
     CHK(mbedtls_ctr_drbg_seed(&pTlsSession->ctrDrbg, mbedtls_entropy_func, &pTlsSession->entropy, NULL, 0) == 0, STATUS_CREATE_SSL_FAILED);
-    CHK(mbedtls_x509_crt_parse_file(&pTlsSession->cacert, KVS_CA_CERT_PATH) == 0, STATUS_INVALID_CA_CERT_PATH);
+
+    // Read and parse CA certificate
+    CHK_STATUS(readFile(KVS_CA_CERT_PATH, FALSE, NULL, &cert_len));
+    cert_buf = (PBYTE) MEMCALLOC(1, cert_len + 1);
```

Comment:
`    CHK(certLength > 0, STATUS_INVALID_CA_CERT_SIZE);`

Reply:
`STATUS_INVALID_CA_CERT_SIZE` error define is not present. I can use `STATUS_INVALID_CERT_PATH_LENGTH` or `STATUS_FORMAT_ERROR` if it's okay.

Reply:
Sure, STATUS_INVALID_CERT_PATH_LENGTH works.